src/reaper_osara.cpp

```diff
@@ -1242,7 +1242,11 @@ bool maybeSwitchToFxPluginWindow() {
 // Handle keyboard keys which can't be bound to actions.
 // REAPER's "accelerator" hook isn't enough because it doesn't get called in some windows.
 LRESULT CALLBACK keyboardHookProc(int code, WPARAM wParam, LPARAM lParam) {
-	if (code != HC_ACTION && wParam != VK_APPS && wParam != VK_RETURN && wParam != VK_F6) {
+	if (code != HC_ACTION){
+		// Return early if this event does not have key code info
```

Comment:
This comment is not strictly correct. `code` can also be `HC_NOREMOVE`, which still provides key code info but isn't something we want to intercept.

Reply:
I'm going to fix the comment if we decide to continue this pull request.